301-drupal8-vmss-glusterfs-mysql/azuredeploy.json

```diff
@@ -125,35 +126,35 @@
     "mysqlUserPassword": {
       "type": "securestring",
       "metadata": {
-        "description": "mysql user password"
+        "description": "mysql user password. For existing enter the existing password. In case of new MySQL server this will be the password for MySqL admin user"
       }
     },
     
-    "mySqlDnsName": {
+    "newMySqlDnsName": {
```

Comment:
This should be called a newMySqlDnsLabelPrefix to set the right expectations. See #966